src/github_text_embed/index.js

```diff
@@ -7,6 +7,8 @@ module.exports = client => {
     const words = msg.content.split(' ')
     const link = words.find(word => word.includes('https://github.com'))
     if (link == null) return
+    const ignoreLink = const link = words.find(word => word.includes('<https://github.com'))
```

Comment:
const ignoreLink = const link = … ? what does that mean?

Reply:
> const ignoreLink = const link = … ? what does that mean?
ignoreLink is ignoring the link that has a prefix on it
I was tempted to use a regex to sort through it, but I didn't know if you wanted that.
```js
const regex = /<http:\/\/github\.com.+>/
```

Reply:
I understand how you did it, why did you use the equal sign two times in that line, this line of code doenst make sense

Reply:
Ah, that was my bad lol.